.travis.yml

```diff
@@ -12,4 +12,5 @@ matrix:
       install:
         - pip install -r event_listener/requirements.txt
       script:
+        - pytest --cov=event_listener/dfusion_db/ --cov-report term-missing
         - mypy event_listener/dfusion_db/ --ignore-missing-imports --strict
```

Comment:
I don't see any reason to ignore missing imports. Why not just remove unused imports.

Reply:
This is for type checking, otherwise it will complain that untyped dependencies cannot be checked. Also this is not really related to this PR.